src/components/common/FormField.tsx

```diff
@@ -37,7 +37,6 @@ const useTextFieldStyles = makeStyles({
 
 const FormField: React.FC<{
   label: string;
-  required?: boolean;
```

Comment:
how come this prop was removed?

Reply:
i'm guessing it was a revert of this comment from before
https://github.com/uwblueprint/paramedics-react/pull/20#discussion_r460352998

Reply:
Joe requested the asterisks get moved to the front of the label, and the required props of the material ui field puts the asterisk at the end of the label so i just removed it and put them directly in the labels